application/routers/fact.py

```
@@ -259,3 +260,44 @@ def search_facts(
     response_class=HTMLResponse,
     include_in_schema=False,
 )
+
+
+def validate_fact_dataset_query_filters(
```

Pylint (4.0.6)

[error] function already defined line 73

(E0102)